frontend/src/App.js

```diff
@@ -182,8 +182,11 @@ class App extends Component {
                 grams: grams,
                 driveEq: json["drive_eq"],
                 totalLandUse: totalLandUse,
-                parkingEq: totalLandUse / 14, // Convert from sq meters to # parking spots
-                totalWaterUse: totalWaterUse * 4.2
+                // parkingEq: totalLandUse / 14, // Convert from sq meters to # parking spots
+                totalTreeUse: totalLandUse / 256, // 256sqft/tree Convert from land use to Californian tree use
+                totalWaterUse: totalWaterUse * 4.2.toExponential(), // Water use in cups
```

Comment:
I don't think exponential representation is very intuitive to people.

Reply:
Thanks for catching this error. Honestly not sure how this got there, maybe it snuck in from some of VScode’s autocompletion? I’ll be more careful from now on.